src/App.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <div id="app1" style="height:100%; overflow-y:hidden;">
+    <div class="chat" style="height: 100%;">
+      <tabs-panel></tabs-panel>
+      <chat-content></chat-content>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+  import { Component } from 'vue-property-decorator';
+  import Vue from '@/types/vue';
+  import TabsPanel from './components/TabsPanel/TabsPanel.vue';
+  import ChatContent from './components/ChatContent/ChatContent.vue';
+  @Component({
+    components: {
+      TabsPanel,
+      ChatContent,
+    },
+  })
+  export default class App extends Vue {
+    mounted() {
+      this.$store.dispatch('initClient');
+    }
+  }
+</script>
+
+<style lang="scss">
+@import url(https://fonts.googleapis.com/css?family=Didact+Gothic|Open+Sans:400,300,600,700);
+@import url(https://irc.eternagame.org/eterna.css);
```

Comment:
> Also, just grab it from `...` and put it directly in the style, don't bother with an `@import`.
If eterna.css will take effect when we embed the chat in the main site, shouldn't we have the `@import` to simulate what will happen when we embed it?
Or did you mean to put everything in the style, not just what we use?

Reply:
That's not what I meant - if you include eterna.css in the app itself, then when it's included in the main website it will get defined *twice* which could cause conflicts.

Reply:
...and if you put it on the test page, that's more accurate to what'll actually be happening anyways (the app is embedded into a page with those things already set)

Reply:
Oh, I see
But I'll wait with it until I write it, which will probably be after I resolve all of these.